js/pages/myToDoList.js

```diff
@@ -317,7 +317,7 @@ const addEventToDoList = () => {
 }
 
 const renderMainBody = (data, collections, tab) => {
-    let template = `<ul class="questionnaire-module-list">`;
+    let template = `<ul class="questionnaire-module-list" role="list">`;
```

Comment:
Is this role specification necessary? I believe this is already the default role for ul. Relevant Stack Overflow discussion here: https://stackoverflow.com/questions/11131234/the-reason-to-use-role-list-and-role-listitem

Reply:
(same as below, I'd prefer to keep the additional semantic tags in)